scripts/analysis/amwg_table.py

```diff
@@ -224,7 +225,7 @@ def amwg_table(adf):
             #End if
 
             #Load model variable data from file:
-            ds = pf.load_dataset(ts_files)
+            ds = utils.load_dataset(ts_files)
```

Comment:
Just double-checking if importing `plotting_functions` is needed in this script anymore?  If not then I would probably recommend removing the `import plotting_functions as pf` line above.